README.md

```diff
@@ -61,3 +61,26 @@ aws s3 cp s3://bucket/foo.fastq.zstd | \
     siz2fastq -z >(aws s3 cp - s3://bucket/foo_1.fastq.gz) \
                  >(aws s3 cp - s3://bucket/foo_2.fastq.gz) \
 ```
+
+## SIZer
+
+Convert paired short-read FASTQ files to split, interleaved, zstd-compressed (SIZ) format.
+Typically used within a data processing pipeline and with compress-upload.sh.
+
+Presently, input FASTQ files with hard wrapping are not supported.
+```
+Usage: ./SIZer [-p <local_prefix>] <reads_per_chunk> <r1_fastq> <r2_fastq> <s3_output_prefix> <compress_script_path> <max_concurrent_jobs> <zstd_threads>
```

Comment:
Capital letters in command names feel very weird to me.  Is `sizer` not an option?